### DISCUSSION TITLE
Where is .env file?

### DISCUSSION THREAD

**Discussion by sulheim** [SysBioChalmers/Sco-GEM#142] [Q&A]
@edkerk, I tried to run `increaseVersion.py` but it doesn't work for me as `find_dotenv()` doesn' find any .env file... 
Do you have a .env file locally?

**Comment by edkerk** [comment]
Note this text was added to `README.md` under [Contributor](https://github.com/SysBioChalmers/Sco-GEM/#contributor):

```python
pip install -r requirements.txt  # installs all dependencies
touch .env
```

Best not to distribute a `.env` file as it could be customised with your own prefered environment variables. With `touch`, it just makes sure that a file exists, and generates an empty file if not.

**Reply by edkerk** [reply]
But for the first run of `increaseVersion.py`, we'd first have to make `version.txt` and `history.md` files in `master`, as these would then be changed by the function. Will do this later today.

**Reply by edkerk** [reply]
Now done, when PR #143 is merged, `master` is ready for running `increaseVersion.py minor` and releasing v1.3.0.

**Reply by sulheim** [reply]
Sounds good! I guess I just should have read the README me more carefully.
